SuperPong/Assets/Scripts/BallController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BallController : MonoBehaviour
+{
+    private Rigidbody sphere;
+    private int velocityRange = 200;
```

Comment:
Make stuff like this public for playtesting.  Once we find a number we really like, we can set it to private to keep it.